targets/TARGET_STM/TARGET_STM32F2/TARGET_NUCLEO_F207ZG/device/TOOLCHAIN_ARM_STD/startup_stm32f207xx.S

```diff
@@ -404,33 +382,6 @@ HASH_RNG_IRQHandler
                 ENDP
 
                 ALIGN
-
```

Comment:
why are we removing these lines in this startup file?

Reply:
To save memory. It was my impression that they are leftovers from somewhere and actually serve no purpose other than eat up memory. They seem to be used by __user_initial_stackheap which is not used for anything as far as I know. 

In case it would be actually needed for something we have those targets that do not have these kind of definitions. For example startup_stm32F411xe.S or startup_stm32l486xx.S which I know work with MCC. Those targets that do have these sections seem to be under TOOLCHAIN_ARM_MICRO. startup_stm32l031xx.S for example has the "Tailor this value ..." comment left but has other parts removed in similar fashion. In fact, I think this is the only STM target that has these definitions under TOOLCHAIN_ARM_STD.

Reply:
I may be wrong though and maybe they do have some purpose. I just don't know what it is.

Reply:
But if there is no reason I would gladly see them gone. It adds up to 1536 bytes of RAM.